catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchParserImpl.java

```diff
@@ -161,7 +159,7 @@ public void populateSpatial(
     }
 
     if (geometry != null) {
-      checkAndReplace(client, WKT_WRITER.write(geometry), OpenSearchConstants.GEOMETRY, parameters);
+      checkAndReplace(client, new WKTWriter().write(geometry), OpenSearchConstants.GEOMETRY, parameters);
```

Comment:
✏️ Pretty sure you can still use `private static final ThreadLocal<WKTWriter> WKT_WRITER_THREAD_LOCAL = ThreadLocal.withInitial(WKTWriter::new)` instead to have the performance benefit of not repeatedly constructing new WKTWriters.

You use it like `WKT_WRITER_THREAD_LOCAL.get().write(geometry)`. Keeping it how it is should be fine in the test code, though.